### ISSUE TITLE
Jovo Deploy googleAssistant .gactionsrc.yaml was not found

### ISSUE THREAD

**Issue by alvarocardenasc** [jovotech/jovo-framework#1238]
## I'm submitting a...

- [x] Bug report
- [x] Feature request
- [x] Documentation issue or request
- [x] Other... Please describe:

## Expected Behavior

Deploy to google actions.

gactions is configured correctly and logged in.

## Current Behavior

I can build the google action, but cant deploy it.

### Error log
```
🚀 Deploying Conversational Action [stage: dev]
  ✖ Pushing project files
                               
› Message:
›  Command failed: gactions push --consumer jovo-cli
›  [ERROR] Can't find a project root. This may be because (1) ".gactionsrc.yaml" was not found in this or any of the parent folders, or (2) if ".gactionsrc.yaml" was found, but the key "sdkPath" was missing, or (3) if ".gactionsrc.yaml" and manifest.yaml were both not found.
›  [ERROR] can not determine project root

› Module:
›  GoogleAssistantCli
```

These are the folders created with jovo build:platform googleAssistant:
![image](https://user-images.githubusercontent.com/26212386/152695239-dc620505-5114-4fc4-bca3-69116f05a062.png)

What I'm missing?

## Your Environment

* Jovo CLI version used: 4.1.0
* Operating System: 10.15.5

**Comment by alvarocardenasc** [comment]
Help?

**Comment by jrglg** [comment]
Please can you try to manually create `manifest.yaml` file under `build\local\platform.googleAssistant` after build? then run deploy command again.

The content of `manifest.yaml` is:
```
"version": "1.0"

```


This might be related [with an issue I have](https://jovodevs.slack.com/archives/C01RH3TMPFF/p1643307250305279).

**Comment by alvarocardenasc** [comment]
It worked!! I manually created the file manifest.yaml
Just got some warnings but in your doc says that I will have to use the Google Action Console and then retrieving the settings using the 'get' command.

**Comment by jrglg** [comment]
I had a look yesterday, and I think it's a bug introduced in [these changes](https://github.com/jovotech/jovo-framework/blob/v4/latest/platforms/platform-googleassistant/src/cli/hooks/BuildHook.ts).

I'm not sure, but I think that's the reason because I ran yesterday the `gactions init hello-world` command and the manifest file is correctly created. Also because the bug started appearing after those changes.

I will try to fix it, but I have to understand the code first.

It would be nice this to be marked as help-wanted because I consider it's an important bug.

Maybe also move the issue to jovo-framework.

[Another user has the same problem](https://jovodevs.slack.com/archives/C01RH3TMPFF/p1643307250305279).

What do you think @jankoenig ?

**Comment by jrglg** [comment]
This is actually very important to new users.

**Comment by jankoenig** [comment]
Thanks for flagging! This was fixed here #1285
